src/screens/TopicPageScreen/index.js

```diff
@@ -59,7 +58,7 @@ const TopicPageScreen = (props) => {
   const [isInitialLoading, setIsInitialLoading] = React.useState(true);
   const [userId, setUserId] = React.useState('');
   const [topicId, setTopicId] = React.useState('');
-  const [isFollow, setIsFollow] = React.useState(false);
+  const [isFollow, setIsFollow] = React.useState(params.isFollowing);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initial state of `isFollow` is now set to `params.isFollowing` instead of `false`. Ensure that `params.isFollowing` is always defined and is a boolean to avoid unexpected behavior.
```diff
- const [isFollow, setIsFollow] = React.useState(false);
+ const [isFollow, setIsFollow] = React.useState(params.isFollowing);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->